pkgs/development/mobile/androidenv/ndk-bundle/default.nix

```diff
@@ -11,7 +11,7 @@ deployAndroidPackage {
     patchShebangs .
 
     patch -p1 \
-      --no-backup-if-mismatch < ${./make_standalone_toolchain.py_18.patch}
+      --no-backup-if-mismatch < ${./make_standalone_toolchain.py_18.patch} || true
```

Comment:
Apparently this isn't needed on newer NDKs.

Reply:
I'm glad you are fixing this, I had to fix this one in my fork too: https://github.com/status-im/nixpkgs/commit/6dacca5e